spec/rspec/core/backtrace_formatter_spec.rb

```diff
@@ -112,19 +112,7 @@ def make_backtrace_formatter(exclusion_patterns=nil, inclusion_patterns=nil)
 
         expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq(["./my_spec.rb:5"])
       end
-
-      it "excludes lines from rspec libs by default", :failing_on_appveyor, :if => RSpec::Support::OS.windows? do
-        backtrace = [
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\expectations\\foo.rb:37",
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\matchers\\foo.rb:37",
-          ".\\my_spec.rb:5",
-          "\\path\\to\\rspec-mocks\\lib\\rspec\\mocks\\foo.rb:37",
-          "\\path\\to\\rspec-core\\lib\\rspec\\core\\foo.rb:37"
-        ]
-
-        expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq([".\\my_spec.rb:5"])
-      end
-
+      
```

Comment:
Okay. My git philosophy is basically the opposite (probably due to personal philosophies more than analysis of actual future usefulness), but this isn't my project, so I'll do it your way. By "place at the head of the history" I did mean rebase, or more generally, something which demultiplexes the history.